src/imgkit/__init__.py

```diff
@@ -1,5 +1,6 @@
 # -*- coding: utf-8 -*-
-"""Wkhtmltopdf python wrapper to convert html to image using the webkit rendering engine and qt"""
+""" Wkhtmltopdf python wrapper to convert html to image using the webkit
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>D209 Multi-line docstring closing quotes should be on a separate line